src/main/java/app/studnicki/ox/Main.java

```diff
@@ -0,0 +1,13 @@
+package app.studnicki.ox;
+
+import app.studnicki.ox.ui.UserInterface;
+import app.studnicki.ox.ui.UserInterfaceFactory;
+
+public class Main {
+  public static void main(String[] args) {
+    UserInterface ui = UserInterfaceFactory.console();
+    Menu menu = new Menu(ui);
+    menu.run();
```

Comment:
Could be done as one line: `new Menu(UserInterfaceFactory.console()).run();`
Since you don't need to reuse vars, why not do it like that?